shared/message/input/Sensors.proto

```diff
@@ -146,12 +146,12 @@ message Sensors {
     /// A set of transforms; one for each frame (All the joints, cameras and base of feet)
     repeated mat4 Htx = 12 [(array_size) = 24];
 
```

Comment:
Threw me for a loop, unsure if the convention isn't followed or the comment is back to front. Reviewer please confirm

Reply:
lgtm